src/test/java/seedu/calculator/calculatorTest.java

```diff
@@ -0,0 +1,50 @@
+package seedu.calculator;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Maybe you can consider using a specific import rather than a wildcard import.
Also perhaps you can consider using the three-part name format for the naming the testing methods: methodUnderTest_inputGiven_expectedOutput

Reply:
Ok, I fixed them in v0.6